forwarder/parse_commandline_args.py

```diff
@@ -108,6 +108,13 @@ def parse_args():
         help="<host:port> Address to the Grafana (Carbon) metrics server",
         type=str,
     )
+    parser.add_argument(
+        "--statistics-update-interval",
+        required=False,
+        help="Update interval (in seconds) to send statistics to Grafana metrics server",
+        type=int,
```

Comment:
why not float? For example 0.5 seconds.

Reply:
In StatisticsReporter class it is `int` so wanted to stay consistent.